e2e/specs/translation-builder-workflows.spec.ts

```diff
@@ -0,0 +1,178 @@
+    });
+
+    await test.step('Then the "Untranslated" translations tab should be selected', async () => {
+      await expect(formBuilderPage.untranslatedTab()).toHaveAttribute('aria-selected', 'true');
+    });
+
+    await test.step('And I should see at least 1 "untranslated" entry', async () => {
+      await expect(page.locator('[data-status="untranslated"]').first()).toBeVisible();
+    });
+
+    await test.step('When I search translations for "Visit Details"', async () => {
+      const searchInput = formBuilderPage.translationSearchInput();
+      await searchInput.fill('Visit Details');
+    });
+
+    await test.step('Then the translation search input should contain "Visit Details"', async () => {
+      const searchInput = formBuilderPage.translationSearchInput();
+      await expect(searchInput).toHaveValue('Visit Details');
+    });
+
+    await test.step('And I should see at least 1 translation result', async () => {
+      const results = page.locator('[data-testid="translation-entry"]');
+      await expect(results).not.toHaveCount(0);
+    });
+
+    await test.step('And the first translation result should be visible', async () => {
+      const results = page.locator('[data-testid="translation-entry"]');
+      await expect(results.first()).toBeVisible();
+    });
+  });
+
+  test('Edit and update an individual translation', async ({ page }) => {
+    const formBuilderPage = new FormBuilderPage(page);
+
+    await test.step('When I open a form in the Translation Builder', async () => {
+      await formBuilderPage.gotoFormBuilder();
+      await formBuilderPage.searchForForm(formDetails.name);
+      await formBuilderPage.page.getByRole('row', { name: formDetails.name }).getByLabel('Edit Schema').first().click();
+    });
+
+    await test.step('And then I go to the translation builder', async () => {
+      await formBuilderPage.translationBuilderTab().click();
+    });
+
+    await test.step('And then I select the French language', async () => {
+      await formBuilderPage.languageDropdown().click();
+      const dropdownMenu = formBuilderPage.translationBuilderPanel().locator('.cds--list-box__menu');
+      await expect(dropdownMenu).toBeVisible();
+      await formBuilderPage.page.getByRole('option', { name: 'French (fr)' }).click();
+    });
+
+    await test.step('And then I edit a translation string and save it', async () => {
+      await formBuilderPage.editTranslationButton(0).click();
+      const translationInput = formBuilderPage.translationValueInput();
+      await translationInput.fill('Test Translation Updated');
+    });
+    await test.step('And then when I preview the form in the language, I should see the edited translation strings', async () => {
+      await formBuilderPage.saveTranslationButton().click();
+      await expect(formBuilderPage.translationModal()).toBeHidden();
+      await expect(formBuilderPage.editTranslationButton(0)).toBeEnabled();
+    });
+  });
```

Comment:
nit: Also missing new line, lets fix this everywhere
```suggestion
  });
  
```